### ISSUE TITLE
Sofort/Klarna Payment implementation uses deprecated Stripe API

### ISSUE THREAD

**Issue by TimOetting** [wagnerwagner/merx-starterkit#14]
The current Sofort/Klarna gateway uses the depracated Sources API.

I get this error while trying to make a Klarna payment:

```
{
    "status": "error",
    "message": "Die Bezahlung konnte nicht initialisiert werden.",
    "code": 500,
    "exception": "Kirby\\Exception\\Exception",
    "key": "error.merx.initializePayment",
    "file": "\/site\/plugins\/merx\/src\/Merx.php",
    "line": 256,
    "details": {
        "message": "The Sources API has been deprecated, and we are no longer allowing new integrations. Please reference the following support article to learn how to update your integration: https:\/\/support.stripe.com\/questions\/i-received-a-sources-api-error-how-should-i-handle-it",
        "code": 0,
        "file": "[...]\/site\/plugins\/merx\/vendor\/stripe\/stripe-php\/lib\/Exception\/ApiErrorException.php",
        "line": 38
    },
    "route": "shop\/checkout"
}
```

**Comment by tobiasfabian** [comment]
Hi Tim,
thanks for your issue.
We already got a fix for the deprecated API, it looks, like we forgot to release the Merx version with the fix. 🤦‍♂️

You can take a look/install [Merx 1.9.0-beta.1](https://github.com/wagnerwagner/merx/releases/tag/v1.9.0-beta.1), which solves your issue.

**Comment by tobiasfabian** [comment]
I published [Merx 1.9.0](https://github.com/wagnerwagner/merx/releases/tag/v1.9.0) and updated the Starterkit.
